### PR TITLE
bring issues the community can help on up to the top

### PR DESCRIPTION
**Pull request by salsakran**
How do these look?

### CODE REVIEW

**Comment by agilliland** [comment]
For me the label `Help Wanted` is pretty ambiguous because it's entirely unclear what kind of help.  I could see that label applying to a lot of different scenarios.

If the intention is for that label to indicate that a developer is needed to accept the issue and begin coding then I'd tweak the label a bit to make that a little more clear.  Something like `Coding Needed` or `Developer Needed` would be a bit more clear and would also follow more naturally from the existing label of `Design Needed`.

**Comment by kdoh** [comment]
FWIW Help Wanted seems to be a pretty standard convention in OSS and especially in the context of Github (which skews towards code contributions). In an idea world it seems like the onus of what the task is should be on the issue itself rather than the label. If we need design help or coding help "help wanted" would apply in both cases and the context of the issue should ideally be enough to help a potential contributor understand.

**Comment by kdoh** [comment]
I'd also add since we're in the situation of having a core team actively working, something like Help Wanted or Loking for contributions might help disambiguate our own process from those looking for places to help chip in.

File: docs/contributing.md [hunk 1]
```diff
@@ -31,13 +31,12 @@ Once a feature has been defined, typically it will be taken on by a product desi
...
 
 ### D) Build the feature
 
-Once a feature is tagged [Feature/Designed](https://github.com/metabase/metabase/labels/Feature%2FDesigned), it is considered ready to be built. A core team member (or you, awesomely helpful person that you are) can start working on it.
-Once once or more people have started to work on a feature, it should be marked 
-[Feature/In Progress](https://github.com/metabase/metabase/labels/Feature%2FIn%20Progress). Once there is a branch+some code, a pull request is opened, linked to the feature + any issues that were pulled together to inform the feature. 
+Once a feature is tagged [Help Wanted](https://github.com/metabase/metabase/labels/Help%20Wanted), it is considered ready to be built. A core team member (or you, awesomely helpful person that you are) can start working on it.
+Once once or more people have started to work on a feature, it should be marked [In Progress](https://github.com/metabase/metabase/labels/In%20Progress). Once there is a branch+some code, a pull request is opened, linked to the feature + any issues that were pulled together to inform the feature.
```
Comment (tlrobinson) [inline docs/contributing.md]: s/once/one/

**Pull request merged**